core/src/main/scala/codesearch/core/search/Search.scala

```diff
@@ -73,9 +73,9 @@ trait Search {
       if (request.spaceInsensitive) SpaceInsensitive.spaceInsensitiveString(preciseMatch) else preciseMatch
     }
     if (request.insensitive) {
-      List("csearch", "-n", "-i", "-f", forExtensions, query)
+      List("csearch", "-n", "-i", "-f", forExtensions, query, request.filter)
```

Comment:
Query should have only one filter field? Maybe we could have a list of additional filters.

Reply:
Yes, Only one filter parameter.